_includes/pbc/all/install-features/202212.0/install-the-dynamic-multiple-stores.md

```diff
@@ -0,0 +1,2125 @@
+| spryker/currency-gui                          | vendor/spryker/currency-gui                          |
+| spryker/locale-data-import                    | vendor/spryker/locale-data-import                    |
+| spryker/locale-gui                            | vendor/spryker/locale-gui                            |
+| spryker/product-category-filter-gui-extension | vendor/spryker/product-category-filter-gui-extension |
+| spryker/store-context-gui                     | vendor/spryker/store-context-gui                     |
+| spryker/store-data-import                     | vendor/spryker/store-data-import                     |
+| spryker/store-extension                       | vendor/spryker/store-extension                       |
+| spryker/store-gui-extension                   | vendor/spryker/store-gui-extension                   |
+| spryker/store-storage                         | vendor/spryker/store-storage                         |
+| spryker-shop/store-widget                     | vendor/spryker-shop/store-widget                     |
+
+{% endinfo_block %}
+
+
+{% info_block infoBox "Note" %}
+
+More details about the updated modules can be found at the [link](https://api.release.spryker.com/release-group/0000). <!-- Update link after release-->
+
+{% endinfo_block %}
+
+
+### Set up database schema and transfer objects
+
+Apply database changes and generate entity and transfer changes:
+    
+```bash 
+console propel:install
+console transfer:generate
+
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied by checking your database:
+
+| DATABASE ENTITY                       | TYPE   | EVENT   |
+|---------------------------------------|--------|---------|
+| spy_store.fk_currency                 | column | added   |
+| spy_store.fk_locale                   | column | added   |
+| spy_country_store                     | table  | added   |
+| spy_locale_store                      | table  | added   |
+| spy_currency_store                    | table  | added   |
+
+Make sure that the following changes have been applied in transfer objects:
+
+| TRANSFER | TYPE | EVENT | PATH |
+| --- | --- | --- | --- |
+| CurrencyCriteria | class | created | src/Generated/Shared/Transfer/CurrencyCriteriaTransfer  |
+| LocaleConditions | class | created | src/Generated/Shared/Transfer/LocaleConditionsTransfer  |
+| SearchContext.storeName | property | added | src/Generated/Shared/Transfer/SearchContextTransfer |
+| SchedulerJob.region     | property | added | src/Generated/Shared/Transfer/SchedulerJobTransfer  |
+| ProductConcrete.stores            | property | added | src/Generated/Shared/Transfer/ProductConcreteTransfer  |
+| Customer.storeName                | property | added | src/Generated/Shared/Transfer/CustomerTransfer  |
+| Store.defaultCurrencyIsoCode      | property | added | src/Generated/Shared/Transfer/CustomerTransfer  |
+
+{% endinfo_block %}
+
+ 
+### Change configuration 
```

Comment:
Please check the structure, it should the same as in https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides
Some examples (This is not the full list, please read the [RFC](https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides) carefully):
1.  Only the main sections should be h1, and internal subsections should have a smaller size of the font.
2.  Section name should be exactly the same.
3.  Section order should be exactly the same.
4.  No other section should be defined.
5.  Each piece of code should have a verification step.
6.  Each section that has plugins/configurations should have a table with namespaces and descriptions. (See example in the link below)
7.  Spryker-shop stuff is a separate part of IG `Install Feature Frontend`. (See example in [here](https://spryker.atlassian.net/wiki/spaces/DOCS/pages/925599310/Quotation+Process+Feature+Integration+-+ongoing))
8. For GLUE we have standalone integration guides, it should not be part of this one.
**9. This IG should be split into several**
    - `Dynamic Store Feature Integration` - the main one, with the changes that were done to `SprykerCore` feature.
    - `Glue Api: Dynamic Store Feature Integration` - those IGs lyes in a different folder.
    -  Connector IGs for each touched feature, e.g. `Prices + Dynamic Store Feature Integration` , `Product + Dynamic Store Feature Integration`, **e.t.c.** (To determine if the change is related to specific feature use ModuleToFeature mapping from Release Group https://release.spryker.com/features)

You can take https://docs.spryker.com/docs/pbc/all/request-for-quote/202212.0/install-and-upgrade/install-features/install-the-quotation-process-feature.html as an example of how IG should be organized and split, for the formatting the main reference should be https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides